Rubberduck.UnitTesting/UnitTesting/TestEngine.cs

```diff
@@ -258,7 +258,7 @@ private void RunWhileSuspended(IEnumerable<TestMethod> tests)
                             OnTestStarted(test);
 
                             // no need to run setup/teardown for ignored tests
-                            if (test.Declaration.Annotations.Any(a => a.AnnotationType == AnnotationType.IgnoreTest))
+                            if (test.Declaration.Annotations.OfType<IgnoreTestAnnotation>().Any())
                             {
```

Comment:
Everywhere else, you use the equivalent of  `Any(annotation => annotation is IgnoreTestAnnotation)`.

Reply:
This is actually a rather subtle bug. None of the annotations attached to a declaration are an `IAnnotation`. Fixed both cases pointed out here